src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -4223,6 +4231,16 @@ namespace HeatBalanceAirManager {
 
     // END Algorithm Section of the Module
 
+    void initializeForExternalHVACManager(EnergyPlusData &state) {
```

Comment:
Eventually this function will be adapted into a whole suite of functions that perform initializations and cleanups and what-not in order to get our built-in HVAC calculations working well with an external HVAC manager.